modules/monitoring-enabling-a-separate-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-a-separate-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling a separate Alertmanager instance for user-defined projects
+ 
+In some clusters, you might not want to use the default platform instance of Alertmanager for user-defined alerts. 
+In these cases, you can optionally enable a dedicated instance of Alertmanager to send alert notifications for user-defined projects in a cluster.
+
+Enabling a separate Alertmanager instance dedicated to user-defined projects offers these benefits:
+
+* Developers and other users of user-defined projects can create user-defined alerts and alert routing without the help of an administrator.
+* A dedicated Alertmanager instance for user-defined projects can help reduce the load on the default platform Alertmanager instance.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have enabled monitoring for user-defined projects in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `user-workload-monitoring-config` `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
++
+. Add `enabled: true` and `enableAlertmanagerConfig: true` under `alertmanager` under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+    alertmanager: 
+      enabled: true <1>
+      enableAlertmanagerConfig: true <2>
+----
+<1> Set the `enabled` value to `true` to enable a dedicated instance of the Alertmanager for user-defined projects in a cluster. Set the value to `false` or omit the key entirely to disable the Alertmanager for user-defined projects.
+<2> Set the `enableAlertmanagerConfig` value to `true` to allow users to create user-defined alerts.
```

Comment:
Set the `enableAlertmanagerConfig` value to `true` to allow users to create user-defined alert routing configurations that use the dedicated platform instance of Alertmanager.